source/numerics.tex

```diff
@@ -1404,14 +1404,14 @@
     class philox_engine;
 
   // \ref{rand.predef}, engines and engine adaptors with predefined parameters
-  using minstd_rand0  = @\seebelow@;
-  using minstd_rand   = @\seebelow@;
-  using mt19937       = @\seebelow@;
-  using mt19937_64    = @\seebelow@;
-  using ranlux24_base = @\seebelow@;
-  using ranlux48_base = @\seebelow@;
-  using ranlux24      = @\seebelow@;
-  using ranlux48      = @\seebelow@;
+  using minstd_rand0  = @\seebelow@;      // freestanding
+  using minstd_rand   = @\seebelow@;      // freestanding
+  using mt19937       = @\seebelow@;      // freestanding
+  using mt19937_64    = @\seebelow@;      // freestanding
+  using ranlux24_base = @\seebelow@;      // freestanding
+  using ranlux48_base = @\seebelow@;      // freestanding
+  using ranlux24      = @\seebelow@;      // freestanding
+  using ranlux48      = @\seebelow@;      // freestanding
   using knuth_b       = @\seebelow@;
   using philox4x32    = @\seebelow@;
```

Comment:
No change requested. It seems to me that Philox engines are also suitable for freestanding, perhaps there should be a follow-up LWG issue to make them freestanding.

Reply:
Please file an issue. This sounds entirely reasonable, but can't be part of this change. Thanks!